### ISSUE TITLE
Can't uninstall from my Mac Mini

### ISSUE THREAD

**Issue by normprojects** [bitgapp/eqMac#462]
Hi,

I tried to uninstall throught its UI or any other uninstaller but still it exists in the system. 

How can I delete this app?

Thanks.

**Comment by Some-T** [comment]
I want to know this also. Does not have a very straightforward uninstall process lol

**Comment by Some-T** [comment]
@alidinc4490 so I did a lot of testing. I tried deleting the app from applications and reinstalling different versions, then letting the uninstaller run for about ten mins to no avail. Next I looked around but on the homepage of this it states:
`sudo rm -rf /Library/Audio/Plug-Ins/HAL/eqMac.driver/ && sudo launchctl kickstart -k system/com.apple.audio.coreaudiod &>/dev/null` to run this command in your terminal. I did this and its completely wiped the sound driver. Then as for the eqMac program itself I simply deleted it to bin from the Applications folder. If you look here to back up what I am saying: https://eqmac.app then click at the bottom how to uninstall.
